common/config/rush/common-versions.json

```diff
@@ -53,6 +53,14 @@
     "@azure/event-hubs": ["^2.1.4"],
 
     // Allow storage-blob-changefeed and storage-file-datalake to use the preview version of storage-blob.
-    "@azure/storage-blob": ["^12.2.0-preview.1"]
+    "@azure/storage-blob": ["^12.2.0-preview.1"],
+
+    // Alow newer node types for opentelemetry due to App Insights TLS ingestion requirements
+    "@types/node": ["^14.0.23"],
+    "nock": ["^13.0.0"],
+    "eslint": ["^7.0.0"],
+    "prettier": ["^2.0.0"],
+    "@typescript-eslint/parser": ["^3.0.0"],
+    "@typescript-eslint/eslint-plugin": ["^3.0.0"]
```

Comment:
this feels like a lot of exceptions

Reply:
I've detached from rush for now. I think the only required one is the node types, but you mentioned you had some workaround for that?